packages/ocamlapi_lwt_unix/ocamlapi_lwt_unix.0.0.1/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "2.0"
+maintainer: "stephanos.tsoucas@gmail.com"
+authors: ["Stephanos Tsoucas"]
+license: "Apache-2.0"
+synopsis: "Path-based HTTP request routing for Ocaml"
+homepage: "https://github.com/nosman/Ocamlapi"
+bug-reports: "https://github.com/nosman/Ocamlapi/issues"
+dev-repo: "git+https://github.com/nosman/Ocamlapi.git"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+    ]
+depends: [
+  "ocaml" {>= "4.04"}
+  "dune"
+  "ocamlapi"
+  "cohttp-lwt"
+  "cohttp-lwt-unix"
```

Comment:
it's fine to only explicitly require `cohttp-lwt-unix` and not `cohttp-lwt` as the first one depends on the other.